elytron/src/main/java/org/wildfly/extension/elytron/FileSystemRealmDefinition.java

```diff
@@ -96,21 +105,42 @@
                     .setRestartAllServices()
                     .build();
 
-    static final SimpleAttributeDefinition HASH_ENCODING = new SimpleAttributeDefinitionBuilder(ElytronDescriptionConstants.HASH_ENCODING, ModelType.STRING, true)
-            .setDefaultValue(new ModelNode(BASE64))
-            .setValidator(new StringAllowedValuesValidator(BASE64, HEX))
-            .setAllowExpression(true)
-            .setFlags(AttributeAccess.Flag.RESTART_RESOURCE_SERVICES)
-            .build();
+    static final SimpleAttributeDefinition HASH_ENCODING =
```

Comment:
It would be good to remove the whitespace changes here since you didn't make changes to `HASH_ENCODING` and `HASH_CHARSET` to make it easier to see your actual changes.

Reply:
Reverted